tests/Unit/Domain/Test_ExcisionSphere.cpp

```diff
@@ -149,6 +154,35 @@ void test_error() {
           "The ExcisionSphere must have a radius greater than zero."));
 }
 
+void test_time_dependent_maps() {
+  using TranslationMap = domain::CoordinateMaps::TimeDependent::Translation<3>;
+  PUPable_reg(SINGLE_ARG(
+      domain::CoordinateMap<Frame::Grid, Frame::Inertial, TranslationMap>));
+  std::unique_ptr<domain::CoordinateMapBase<Frame::Grid, Frame::Inertial, 3>>
+      time_dep_map = std::make_unique<
+          domain::CoordinateMap<Frame::Grid, Frame::Inertial, TranslationMap>>(
+          TranslationMap{"DuckDuckGoose"});
+
+  ExcisionSphere<3> excision_sphere{
+      1.0, tnsr::I<double, 3, Frame::Grid>{{0.0, 0.0, 0.0}}, {}};
+
+  CHECK_FALSE(excision_sphere.is_time_dependent());
+  CHECK_THROWS_WITH(excision_sphere.moving_mesh_grid_to_inertial_map(),
+                    Catch::Contains("Trying to access grid to inertial map "
+                                    "from ExcisionSphere, but it has "
+                                    "not been set."));
+
+  excision_sphere.inject_time_dependent_maps(time_dep_map->get_clone());
+  test_serialization(excision_sphere);
+  CHECK(excision_sphere.is_time_dependent());
+  CHECK(excision_sphere.moving_mesh_grid_to_inertial_map() == *time_dep_map);
+
+  const ExcisionSphere<3> copy_assigned_excision_sphere = excision_sphere;
```

Comment:
That's not a copy assignment.  You need to assign to a preexisting object.

Reply:
Oops 😅